invenio_app_rdm/records_ui/utils.py

```diff
@@ -55,3 +56,67 @@ def set_default_value(record_dict, value, path, default_prefix="metadata"):
         path = "{}.{}".format(default_prefix, path)
 
     dict_set(record_dict, path, value)
+
+
+def external_resources(record):
+    """Make number compact if too long."""
+    result = []
+    resources_cfg = current_app.config["APP_RDM_RECORD_LANDING_PAGE_EXTERNAL_LINKS"]
+    for cfg in resources_cfg:
+        res = cfg["render"](record)
+        if res:
+            result.append(res)
+    return [item for sublist in result for item in sublist]
```

Comment:
nit: I somewhat prefer [the `itertools.chain.from_iterable` approach ](https://stackoverflow.com/a/953097), instead of the nested list comprehensions in terms of readability.